src/Compiler/FSComp.txt

```diff
@@ -1738,4 +1738,5 @@ featureReuseSameFieldsInStructUnions,"Share underlying fields in a [<Struct>] di
 3860,chkStaticMembersOnObjectExpressions,"Object expressions cannot implement interfaces with static abstract members or declare static members."
 3861,chkTailCallAttrOnNonRec,"The TailCall attribute should only be applied to recursive functions."
 3862,parsStaticMemberImcompleteSyntax,"Incomplete declaration of a static construct. Use 'static let','static do','static member' or 'static val' for declaration."
-3863,parsExpectingField,"Expecting record field"
\ No newline at end of file
+3863,parsExpectingField,"Expecting record field"
+3864,tooManyMethodsInDotNetTypeWritingAssembly,"The type '%s' has too many methods. Found: '%d', maximum: '%d'"
```

Comment:
Shall we mention if this includes autogenerated methods too?

Reply:
I don't think it's really necessary, if it turns out to be a source of confusion a web page or blog post can take care of it.